core/src/main/scala/com/salesforce/op/stages/impl/feature/MultiPickListMapVectorizer.scala

```diff
@@ -63,9 +68,18 @@ class MultiPickListMapVectorizer[T <: OPMap[Set[String]]]
         k -> cats.map(_ -> 1L).groupBy(_._1).map { case (c, a) => c -> a.map(_._2).sum }
       }
     }
+    implicit val classTag: ClassTag[T#Value] = ReflectionUtils.classTagForWeakTypeTag[T#Value]
```

Comment:
can we put all this under a check that the max cardinality is < 1 so we dont do the extra calculation if we dont want to throw things out?